exercises/D-logical-operators/exercise2.js

```diff
@@ -4,8 +4,39 @@
   This program calls some functions that are either missing or incomplete.
   Update the code so that you get the expected result.
 */
+var num = " ";
 
-function isNegative() {}
+function isNegative(num) {
+    if (num < 0) {
+        return true;
+    } else if (num > 0) {
+        return false;
+    }
+}
+
+function isBetween5and10(num) {
+    if (num > 5 && num <= 10) {
+        return true;
+    }
+    return false;
+}
+
+function isShortName(name) {
+    if (name === "Daniel") {
```

Comment:
Here you want the name to be short, it doesn't necessarily have to be Daniel, so you need to find out the length of the name